.buildkite/pipeline.yml

```diff
@@ -218,6 +223,12 @@ steps:
   - label: ":docker: Publish docker image"
     key: "publish"
     command: ".buildkite/scripts/build_push_docker_image.sh"
+    env:
+      DOCKER_REGISTRY: "docker.elastic.co"
```

Comment:
Is this line necessary? I'm not seeing it on `main`: https://github.com/elastic/fleet-server/blob/bc54df84f657e995777f11ba1c4a502e8fb99f8b/.buildkite/pipeline.yml#L329-L336